app/models/manageiq/providers/openstack/storage_manager/cinder_manager.rb

```diff
@@ -1,4 +1,4 @@
-class ManageIQ::Providers::Openstack::StorageManager::CinderManager < ManageIQ::Providers::StorageManager::CinderManager
+class ManageIQ::Providers::Openstack::StorageManager::CinderManager < ManageIQ::Providers::StorageManager
```

Comment:
darn - this class no longer matches the filename.
Is that an oversight, or do we need it to be this way for class loading?

Reply:
ignore me -- changed the parent class not the main class

Reply:
Still looks like it matches the filename to me...am I missing something?